README.md

```diff
@@ -1,29 +1,127 @@
 # Quarkus Test Suite
 
+The aim of this test suite is to validate Quarkus framework features along the time. Under the hood we are using [Quarkus-test-framework](https://github.com/quarkus-qe/quarkus-test-framework) that we designed for this proposal. 
+
 The test suite includes:
 - Bare metal scenario testing
 - OpenShift scenario testing
+- Native testing
+
+## How-to run Quarkus test suite
+
+Please before start be sure that you have all [prerequisites](#Prerequisites) installed.
+
+### Openshift
 
 For running the tests in OpenShift, it is expected that the user is logged into an OpenShift project:
 - with OpenShift 4, run `oc login https://api.<cluster name>.<domain>:6443`
 
 To verify that you're logged into correct project, you can run `oc whoami` and `oc project`.
 
-Running the tests amounts to standard `mvn clean verify`.
-This will use a specific Quarkus version, which can be modified by setting the `quarkus.platform.version` property.
-Alternatively, you can use `-Dquarkus-core-only` to run the test suite against Quarkus `999-SNAPSHOT`.
-In that case, make sure you have built Quarkus locally prior to running the tests.
-Moreover, all the modules are grouped by types: `http`, `security`, `messaging`... Hence we can run a concrete set of scenarios 
-just by selecting the right profile: `-P http-modules` or `-P security-modules`. Take into account that if you are selecting 
-another profile as OpenShift (`mvn clean verify -Dopenshift`), you need to explicitly to add `-Dall-modules` to include all 
-the modules in all the profiles.
+By default [Quarkus-test-framework](https://github.com/quarkus-qe/quarkus-test-framework) uses [ephemeral namespaces](https://github.com/quarkus-qe/quarkus-test-framework/wiki/6.-Openshift#disable-ephemeral-namespaces) in order to be deterministic and don't populate your OCP environment with random resources. After each scenario, all the instantiated resources will be removed.
+
+Practical Example:
+
+User: `I would like to run http-minimum module.` 
+
+```shell
+mvn clean verify -Dall-modules -Dopenshift -pl http/http-minimum 
+```
+
+**Note:** here we are combining two profiles, `openshift` in order to trigger Openshift execution mode and `all-modules` to enable `http/http-minimum` that is located under `http-modules` or also `all-modules` profile
+
```

Comment:
It would be good to add a note to check profiles defined in pom.xml to see the key testing profiles.

Reply:
I have added a new section called `Profiles overview`, WDYT?